rs/compression/src/elias_fano/ef.rs

```diff
@@ -51,18 +54,20 @@ impl EliasFano {
         Self {
             #[cfg(any(debug_assertions, test))]
             universe,
-            size,
+            num_elem,
             lower_bits,
-            upper_bits: BitVec::with_capacity(2 * size),
+            upper_bits: BitVec::with_capacity(2 * num_elem),
             lower_bit_mask,
             lower_bit_length,
+            cur_high: 0,
+            cur_index: 0,
         }
     }
 
     /// Returns the value at the given index
     #[allow(dead_code)]
```

Comment:
probably useful to note that this function is inefficient, should only be used in tests